README-CN.md

```diff
@@ -1,65 +1,127 @@
 # ob-operator
 
-ob-operator 可以让 OceanBase 以容器的方式，无缝运行在公有云或私有部署的 Kubernetes 集群上。
+ob-operator 是满足 Kubernetes Operator 扩展范式的自动化工具，可以极大简化在 Kubernetes 上部署和管理 OceanBase 集群及相关资源的过程。
 
-ob-operator 现已支持 OceanBase 集群的创建、删除，完整的节点生命周期管理，并通过 Service 的形式暴露给用户使用。后续会支持租户管理、ob-proxy 管理、多 Kubernetes 集群等功能。
+## 快速上手
 
-目前支持通过 YAML、Kustomize 来部署 ob-operator。后续会支持 Helm 形式部署。
+这部分以部署 ob-operator 和 OceanBase 集群为例指导您如何快速使用 ob-operator 进行 OceanBase 集群的管理。
 
-![01](docs/img/01.png)
+### 前提条件
 
-## 环境依赖
+开始之前请准备一套可用的 Kubernetes 集群，并且至少可以分配 2C, 10G 内存以及 100G 存储空间，ob-operator 依赖 [cert-manager](https://cert-manager.io/docs/), cert-manager 的安装可以参考对应的[安装文档](https://cert-manager.io/docs/installation/)，本例子中的 OceanBase 集群存储依赖 [local-path-provisioner](https://github.com/rancher/local-path-provisioner) 提供, 需要提前进行安装。
 
-构建 ob-operator 需要 Go 1.16 版本及以上。
+### 部署 ob-operator
 
-运行 ob-operator 需要 Kubernetes 1.16 版本及以上。
-尚未支持多 Kubernetes 版本，后续会支持。
+通过以下命令即可在 K8s 集群中部署 ob-operator：
 
-## 支持的 OceanBase 版本
+* Stable
+```shell
+kubectl apply -f https://raw.githubusercontent.com/oceanbase/ob-operator/2.1.0_release/deploy/operator.yaml
+```
 
-后续会不断支持新的 OceanBase 社区版本。
-暂不支持 3.1.2 之前的旧的 OceanBase 版本。目前仅对 OceanBase 社区版 3.1.2 进行了测试。
+* Helm chart
```

Comment:
Helm chart is an independent part from deploy using configuration file, move it before Stable or after Development

Reply:
and there needs a title to specify deploy using helm or configuration file